pom.xml

```diff
@@ -1307,6 +1308,7 @@
             <version>${maven-surefire-plugin.version}</version>
             <configuration combine.self="append">
               <skip>${skipUTs}</skip>
+              <trimStackTrace>false</trimStackTrace>
```

Comment:
this is also set in    
<pluginManagement>
      <plugins>
        <plugin>

is it not effective?

Reply:
I think it's not effective for me w/ or w/o this change. During Github CI check I still see truncated stacktrace. Will revert this change